ktor-core/src/org/jetbrains/ktor/features/CallLogging.kt

```diff
@@ -47,12 +51,21 @@ class CallLogging(private val log: Logger, private val monitor: ApplicationMonit
 
     }
 
+    private fun log(message: String) {
+        when (configuration.level) {
+            Level.ERROR -> if (log.isErrorEnabled) log.error(message)
```

Comment:
Looks like it doesn't make any sense to allow `ERROR` or `WARNING` level for call logging. Not sure we should limit it here, but that looks weird.